src/account/AccountStorage.sol

```diff
@@ -47,8 +41,7 @@ struct AccountStorage {
     uint8 initialized;
     bool initializing;
     // Plugin metadata storage
-    EnumerableSet.AddressSet plugins;
-    mapping(address => PluginData) pluginData;
+    EnumerableMap.AddressToUintMap pluginManifestHashes;
```

Comment:
Why not `AddressToBytes32Map`?

Reply:
Unfortunately, it doesn't exist: https://docs.openzeppelin.com/contracts/5.x/api/utils#EnumerableMap

This is the only map that has `address` as a key type.

Reply:
We need to use a newer version!

Reply:
I think it is the newest version, haha

Reply:
Ah that's not released yet, check under the releases tab. I'd prefer to avoid using pre-release content, but we should update this when 5.1 is released.